users/views.py

```diff
@@ -0,0 +1,46 @@
+from django.shortcuts import render
+from django.views import View
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from django.http import Http404
+from users.models import User
+from users.serializers import UserSerializer
+# Create your views here.
+class UserApi(APIView):
+    permission_classes = []
```

Comment:
here you are not adding any new permission or authentication_classes why are leaving them here.
```python
authentication_classes = (TokenAuthentication,)
    permission_classes = [IsAuthenticated]
```